pkg/log/zap/logger.go

```diff
@@ -75,10 +75,16 @@ func getConfig(destWriter io.Writer) config {
 	}
 	if levelVal.set {
 		c.level = zap.NewAtomicLevelAt(levelVal.level)
+
```

Comment:
Nit: this line is extraneous

Reply:
That's not new code though. That was there from before :)

Reply:
Are you sure? This is what it looks like on master (no blank line between setting `c.level` and the closing bracket): https://github.com/operator-framework/operator-sdk/blob/c6e209a42be7d4ef6a5b1077cfafea7fc6f7a77a/pkg/log/zap/logger.go#L76-L78

I don't think we'd see the + green line otherwise. I'm guessing it was just left over from moving the disable sampling logic.

Reply:
Upon further review, that line is NOT extraneous. Without it the level will never be set correctly. It will either be debug or warn. We'd have to change the else block of the development check. But if we're doing that we'd be checking to see if levelVal was set before setting the level. And if it wasn't set we'd default to WarnLevel. At which point we'd just be moving this line to a different place. So with all due respect, I think it should remain here for this PR.